src/tools/compiletest/src/runtest.rs

```diff
@@ -523,7 +518,8 @@ impl<'test> TestCx<'test> {
         let mut rustc = Command::new(&self.config.rustc_path);
 
         let out_dir = self.output_base_name().with_extension("pretty-out");
-        remove_and_create_dir_all(&out_dir);
+        remove_and_create_dir_all(&out_dir)
+            .expect("Failed to remove and recreate output directory");
```

Comment:
Suggestion: include more context on *which* path failed and the underlying I/O error

```suggestion
        remove_and_create_dir_all(&out_dir)
            .unwrap_or_else(|e| panic!("failed to remove and recreate output directory `{out_dir}`: {e}"));
```

Ditto on other sites, but s/output directory/aux dir/ or whatever as contextually suitable.